opsdroid/parsers/tests/test_parser_format.py

```diff
@@ -0,0 +1,81 @@
+"""Test the opsdroid parse_format parser."""
+import asynctest.mock as amock
+
+from opsdroid.cli.start import configure_lang
+from opsdroid.core import OpsDroid
+from opsdroid.matchers import match_parse
+from opsdroid.events import Message
+from opsdroid.parsers.parseformat import parse_format
+
+
+configure_lang({})
+
+
+async def getMockSkill():
+    async def mockedskill(opsdroid, config, message):
+        pass
+
+    mockedskill.config = {}
+    return mockedskill
+
+
+async def test_parse_format_match_condition():
```

Comment:
We have an opsdroid fixture, so we can refactor this text a bit - we can remove the `with` statement and add `opsdroid` as arg for the test.

```suggestion
async def test_parse_format_match_condition(opsdroid):
```

Reply:
Hi @FabioRosado I already tried using opsdroid as an argument but I kept getting an error about it not being a fixture. This is strange since it worked as a fixture for the test_web.py. So I am not sure if I am missing an import or if I am just incorrectly using it.